src/tls_wrap.h

```diff
@@ -171,11 +157,10 @@ class TLSWrap : public AsyncWrap,
   StreamBase* stream_;
   BIO* enc_in_;
   BIO* enc_out_;
-  crypto::NodeBIO* clear_in_;
+  std::vector<uv_buf_t> pending_cleartext_input_;
```

Comment:
Would a std::list be more appropriate here?

Reply:
@TimothyGu Why? `uv_buf_t` is a pretty small structure, so what I would guess is that the overhead of possibly having a small number of them being unused is more or less negligible compared to allocating a list node for every buffer?

Reply:
based on the variations I tried with http2, the performance difference is inconsequential. Using `std::vector` here is just fine.